nunit.sln

```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```

Comment:
Me too. But of course my real plan is to just move out. Then I can drop windows entirely and stick with Linux. :-)